app/jinja_filters.py

```diff
@@ -1,23 +1,18 @@
 # coding: utf-8
 
+import flask
 import re
 import string
 
 from datetime import datetime
-
-import flask
-
 from jinja2 import Markup, escape, evalcontextfilter
 
 blueprint = flask.Blueprint('filters', __name__)
 
 
 @blueprint.app_template_filter()
 def format_currency(value):
-    if value is not None and len(str(value)) > 0:
-        return "£{:,}".format(value)
-    else:
-        return ""
+    return "£{:,.2f}".format(value)
```

Comment:
Why has the if statement been removed? If a None value is passed to format_currency a TypeError exception will  now be thrown. You could have just added the .2f to the existing return.